### ISSUE TITLE
F0905 11:26:43.151420 12241 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Video_Data

### ISSUE THREAD

**Issue by awesome-yz** [facebook/C3D#451]
I have caffe version 1.0.0 installed and it compiled correctly with video_data_layer included. When I run c3d_resnet18_sports1m_r2_iter_2800000.caffemodel for feature extraction I get the error for Video Data layer, which is included in the prototxt file.  The error trace is:

F0905 11:26:43.151420 12241 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: VideoData (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, BlackHole, Concat, ContrastiveLoss, Convolution, Convolution3D, Crop, Data, Deconvolution, Deconvolution3D, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, InfogainLoss, InnerProduct, Input, L2Normalize, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Pooling3D, Power, Python, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile)
*** Check failure stack trace: ***
    @     0x7fd85fc320cd  google::LogMessage::Fail()
    @     0x7fd85fc33f33  google::LogMessage::SendToLog()
    @     0x7fd85fc31c28  google::LogMessage::Flush()
    @     0x7fd85fc34999  google::LogMessageFatal::~LogMessageFatal()
    @     0x7fd86004d3b4  caffe::Net<>::Init()
    @     0x7fd86004ef37  caffe::Net<>::Net()
    @     0x55ece66bb679  feature_extraction_pipeline<>()
    @     0x7fd85f2b5b97  __libc_start_main
    @     0x55ece66badda  _start
Aborted (core dumped)

Please help me resolve this issue, I've been at it for days. Thank you.
